docs/source/using/profiles.adoc

```diff
@@ -63,4 +85,33 @@ To delete a profile, run:
 $ minishift profile delete PROFILE_NAME
 ----
 
-NOTE: The default profile 'minishift'  can not be deleted.
+NOTE: The default profile *minishift* cannot be deleted.
+
+[[example-workflows-profiles]]
+== Example Workflows
+
+To create a new profile *foo* with specific configurations, run:
+
+----
+$ minishift profile set foo
+$ minishift config set memory 4GB
+$ minishift config set cpus 2
+$ minishift addon enable anyuid
+$ minishift start
+----
+
+Alternate method to create the profile *foo*:
```

Comment:
The alternative is to execute a series of commands each specifying the targeted profile explicitly using the --profile flag